src/keys.hh

```diff
@@ -21,15 +21,16 @@ struct Key
         Alt     = 1 << 1,
         Shift   = 1 << 2,
 
-        MousePress   = 1 << 3,
-        MouseRelease = 1 << 4,
-        MousePos     = 1 << 5,
-        MouseWheelDown = 1 << 6,
-        MouseWheelUp = 1 << 7,
-        MouseEvent = MousePress | MouseRelease | MousePos |
-                     MouseWheelDown | MouseWheelUp,
-
-        Resize = 1 << 8,
```

Comment:
Note that `Resize` used to be `1 << 8` while the next entry `MenuSelect` is `1 << 10` - was that done on purpose, or was it simply an artifact?